client/src/routes/routs.tsx

```diff
@@ -0,0 +1,30 @@
+import {createBrowserRouter, Navigate} from "react-router-dom";
+import {RegistrationPage} from "./RegistrationPage";
+import {HomePage} from "./HomePage";
+import {App} from "../App";
+import { CalendarPage } from "./CalendarPage";
+
+export const router = createBrowserRouter([
```

Comment:
I suggest we stick with classic JSX "components" for routes. Use Routes, Route etc.